java/arcs/core/data/proto/policy.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+
+package arcs;
+
+option java_package = "arcs.core.data.proto";
+option java_multiple_files = true;
+
+// Defines a data usage policy.
+message PolicyProto {
+  // Policy name.
+  string name = 1;
+
+  // Targets governed by the policy.
+  repeated PolicyTargetProto targets = 2;
+
+  // Additional config metadata for the policy.
+  repeated PolicyConfigProto configs = 3;
+
+  // Human readable description for the policy.
+  string description = 4;
+
+  enum EgressType {
+    EGRESS_TYPE_UNSPECIFIED = 0;
+    LOGGING = 1;
+    FEDERATED_AGGREGATION = 2;
+  }
+  // Egress type permitted by the policy.
+  EgressType egress_type = 5;
+}
+
+message PolicyTargetProto {
+  // Schema type of the store targeted by the policy.
+  string schema_type = 1;
+
+  // The maximum age of the data that can be used, as a duration (not timestamp)
+  // in milliseconds.
+  int64 max_age_ms = 2;
```

Comment:
should we allow string representation here, like in the manifests: `max_age: 3 days`? we could reuse the same parsing code for it, and it's more readable. (could always use a comment of course, but..)

Reply:
I'd prefer to get all of the parsing out of the way in the one place (that is, the Arcs manifest parser is responsible for parsing and converting "3 days" -> 295200000 ms, from there onwards we talk only in ms)

Reply:
I guess that's ok for policies (i didn't do it this way for recipes, because we need to serialize them back to manifest, and it's more readable with the original units)